src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesTabItem.tsx

```diff
@@ -15,10 +17,15 @@ const StakesItems: FC<StakesProps> = ({
   status,
 }) => {
   return (
-    <li className="flex flex-col py-4 border-b-[0.0625rem] border-gray-100">
+    <li
+      className={clsx(
+        styles.stakesItem,
+        'first:pt-2 last:pb-6 sm:first:pt-0 sm:last:pb-1.5',
```

Comment:
Can't this styles be also in `StakesTabItem.module.css`?

Reply:
Moved